src/ogd/games/BLOOM/schemas/BLOOM.json.template

```diff
@@ -988,36 +988,96 @@
       },
       "aggregate":{
          "ActiveTime": {
-             "threshold": 30,
-             "enabled": true,
-             "type": "ActiveTime",
-             "description": "Active time of a player",
-             "return_type": "timedelta"
+            "threshold": 30,
+            "enabled": true,
+            "type": "ActiveTime",
+            "description": "Active time of a player",
+            "return_type": "timedelta"
+         },
+         "AverageActiveTime": {
+            "enabled": true,
+            "type": "AverageActiveTime",
+            "description": "Average active time of a player",
+            "return_type": "timedelta"
+         },
+         "BloomAlertCount": {
+            "enabled": true,
+            "type": "BloomAlertCount",
+            "description": "Count of bloom alerts",
+            "return_type": "int"
+         },
+         "BuildCount": {
+            "enabled": true,
+            "type": "BuildCount",
+            "description": "Count of builds",
+            "return_type": "int"
+         },
+         "BuildingUnlockCount": {
+            "enabled": true,
+            "type": "BuildingUnlockCount",
+            "description": "Count of building unlocks",
+            "return_type": "int"
+         },
+         "CountyBloomAlertCount": {
```

Comment:
These per-county features need to be moved to the `per_count` section, like we did with your local copy of `BLOOM.json`